polishNotationCalc.cpp

```diff
@@ -0,0 +1,92 @@
+#include "polishNotationCalc.hpp"
+namespace dev
+{
+    PolishCalc::PolishCalc()
+    {
+        insertFunctionInsideMap();
+    }
+
+    bool PolishCalc::isNumeric(const char num)
+    {
+        double temp = num - '0';
+        if (temp >= 0 && temp <= 9)
+            return true;
+
+        return false;
+    }
+    bool PolishCalc::isOperator(const char num)
+    {
+        if (num == '+' || num == '-' || num == '*' || num == '/')
+            return true;
+        return false;
+    }
+   
+    double PolishCalc::calc( const std::string &str)
+    {
+        if (str.size() == 0)
+            throw Exception{"Nothing to work on as Input is empty"};
+        for (int i = 0; i < str.size(); i++)
+        {
+            if (str[i] == ' ')
+                continue;
+            if (isNumeric(str[i]))
+            {
+                pushOperandInStack(str, i);
```

Comment:
Try to work with some kind of tokens but not charters. 
It is not obvious that pushOperandInStack will modify `i` variable. If it is possible. Try to avoid it